src/main/resources/test_files/runtime/ComparisonValue.iq

```diff
@@ -0,0 +1,13 @@
+(:JIQS: ShouldRun; Output="( true, false, true, false, true, false, true, false, true, false, true, false)" :)
+2 eq 2,
+2 eq 3,
+2 ne 3,
+2 ne 2,
+2 lt 3,
+3 lt 2,
+3 gt 2,
+2 gt 3,
+2 le 2,
+2 le 1,
+2 ge 2,
+1 ge 2
```

Comment:
We also need tests with the empty sequence: on the left, on the right, and on both sides.
We also need tests that produce type errors (string on one side, number on the other side, and various other combinations of incompatible types)

Reply:
We also need tests for null. Null is only equal to itself, not to anything else (but no error), and is smaller than anything.

Reply:
We also need tests on strings, booleans, between numbers (decimals, doubles, integers etc) as well as with objects/arrays to check that an error is thrown.